02_configure_host.sh

```diff
@@ -68,7 +68,7 @@ fi
 
 export ANSIBLE_FORCE_COLOR=true
 
-if use_podman_registry; then
+if [[ ! -z "${MIRROR_IMAGES}" || $(env | grep "_LOCAL_IMAGE=")  || ! -z "${ENABLE_CBO_TEST:-}" || ! -z "${ENABLE_LOCAL_REGISTRY}" ]]; then
```

Comment:
With this patch when I set
export REGISTRY_BACKEND=quay
I got the error below.

Can you add to this file:
`source oc_mirror.sh`

+(./02_configure_host.sh:72): main(): setup_local_registry
+(utils.sh:534): setup_local_registry(): [[ quay = \p\o\d\m\a\n ]]
+(utils.sh:537): setup_local_registry(): setup_quay_mirror_registry
utils.sh: line 537: setup_quay_mirror_registry: command not found
+(utils.sh:1): setup_local_registry(): removetmp
+(utils.sh:668): removetmp(): '[' -n ' /tmp/test-token--hLZwRQT5DH' ']'
+(utils.sh:668): removetmp(): rm -rf /tmp/test-token--hLZwRQT5DH
make: *** [Makefile:39: configure] Error 127

Reply:
Thanks for detecting it @bfournie! I will for the moment revert back to the usage of the podman registry, which in any case will be sufficient for covering the local builds